docker/worker/Dockerfile

```diff
@@ -27,6 +27,5 @@ RUN chmod +x /root/qemu/*.sh && /root/qemu/kvm-mknod.sh && \
     chmod -R ug+rw /usr/share/openqa/script/worker /var/lib/openqa/cache /var/lib/openqa/pool && \
     find /usr/share/openqa/script/worker /var/lib/openqa/cache /var/lib/openqa/pool -type d -exec chmod ug+x {} \;
 
-USER _openqa-worker
```

Comment:
Why is this now missing?

Reply:
The reason is that if we move the installation of the (zypper) qemu-ovmf-x86_64 or qemu-uefi-aarch64 to the execution we'll need to be superuser to install this. After that the main app will be executed with the user _openqa-worker... This comentary continues in the next comment https://github.com/os-autoinst/openQA/pull/3475#discussion_r506225301